go-controller/pkg/cni/helper_linux.go

```diff
@@ -424,10 +432,10 @@ func ConfigureOVS(ctx context.Context, namespace, podName, hostIfaceName string,
 	}
 
 	if err := waitForPodInterface(ctx, ifInfo, hostIfaceName, ifaceID, podLister, kclient,
-		namespace, podName, initialPodUID); err != nil {
+		pod.Name, pod.Name, initialPodUID); err != nil {
```

Comment:
I think one of these args is wrong. Should be `pod.Namespace, pod.Name`

Reply:
Good catch!